apps/src/code-studio/components/progress/ScriptOverview.jsx

```diff
@@ -173,11 +177,13 @@ class ScriptOverview extends React.Component {
               viewAs={viewAs}
               isRtl={isRtl}
               resources={teacherResources}
+              migratedResources={migratedTeacherResources}
```

Comment:
nit: would it be worth being consistent with the name? either migratedTeacherResources or migratedResources. My vote is for the first

Reply:
That makes sense to me. I'll update